packages/relay/src/lib/clients/mirrorNodeClient.ts

```diff
@@ -105,6 +106,19 @@ export class MirrorNodeClient {
             timeout: 10 * 1000
         });
 
+        //@ts-ignore
+        axiosRetry(axiosClient, {
+            retries: parseInt(process.env.MIRROR_NODE_RETRIES!) || 3,
+            retryDelay: () => {
```

Comment:
Should we consider and exponential backoff strategy?

Reply:
Added.

Reply:
Okay. Start simple, open a ticket for exponential backoff to come back to later

Reply:
Opened ticket https://github.com/hashgraph/hedera-json-rpc-relay/issues/689